lib/whitehall/featured_document_updater.rb

```diff
@@ -0,0 +1,44 @@
+module Whitehall
+  class FeaturedDocumentUpdater
+    attr_reader :document
+
+    def initialize(document)
+      @document = document
+    end
+
+    def call
+      update_all_features(document)
+    end
+
+  private
+
+    def update_all_features(document)
+      features = Feature.where(document_id: document.id)
+      latest_published_edition = document.editions.latest_published_edition.first
+      return unless latest_published_edition
+
+      public_url = Whitehall.url_maker.public_document_url(latest_published_edition)
+
+      return unless valid_link_type?(latest_published_edition)
+
+      features.each do |feature|
+        offsite_link = OffsiteLink.create(title: latest_published_edition.title,
+                                          summary: latest_published_edition.summary,
+                                          link_type: "content_publisher_#{link_type(latest_published_edition)}",
+                                          url: public_url,
+                                          date: Time.zone.now,
```

Comment:
Do we know what this date is? It seems a bit strange to be setting it to current time so wonder if there's an existing date we should use?

Reply:
Looking at an organisations page this seems to be the public updated at of the content.